### ISSUE TITLE
Can I include this installer in Commercial app？

### ISSUE THREAD

**Issue by Sinsjr2** [dotnet/windowsdesktop#2878]
### Question
I know This repository is The Mit License and this installer ( https://dotnet.microsoft.com/ja-jp/download/dotnet/6.0 ) is multi license,
Can I include this installer in commercial app installer CD?

"dotnet windows" includes Licenses other than The MIT License.
https://github.com/dotnet/core/blob/main/license-information-windows.md
Also, "THIRD-PARTY-NOTICES.txt" in install folder is described many OSS.

As above this installer is multi license, so I'm wondering if it can be inluded in commercial app installer CD.

**Comment by richlander** [comment]
Yes. The main question you want to answer is whether .NET 6 gives you redistributable rights. It does, both for Windows and other supported operating systems.
